### PR TITLE
update recoverable items sub folders to Teams

### PR DESCRIPTION
**Pull request by StefanieBier**
New folder in recoverable items to hold chat messages if edited or deleted. 

https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-teams?view=o365-worldwide#how-retention-works-with-microsoft-teams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/recoverable-items-folder.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ The Recoverable Items folder contains the following subfolders:
 
 - **Calendar Logging**: This subfolder contains calendar changes that occur within a mailbox. This folder isn't available to users.
 
+- **SubstrateHolds**: If In-Place Hold, Litigation Hold, or a Microsoft 365 or Office 365 Teams Chat retention policy is enabled, this subfolder contains the original copy of the teams message if the message has been modified or deleted. A copy of the item before modification is saved. This folder isn't visible to end users.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/recoverable-items-folder/recoverable-items-folder.md:80]:
```suggestion
- **SubstrateHolds**: If In-Place Hold, Litigation Hold, or a Microsoft 365 or Office 365 Teams Chat retention policy is enabled, this subfolder contains the original copy of the Teams message if the message has been modified or deleted. A copy of the item before modification is saved. This folder isn't visible to end users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**